src/common/generalMobilePlatform.ts

```diff
@@ -33,12 +33,24 @@ export class GeneralMobilePlatform {
         return this.remoteExtension.getPackagerPort().then(port => {
             return Packager.isPackagerRunning(Packager.getHostForPort(port))
                 .then(isRunning => {
-                    if (isRunning) {
+                    if (!isRunning) {
+                        Log.logMessage("Starting React Native Packager.");
+                        return this.remoteExtension.startPackager();
+                    }
+
+                    return this.remoteExtension.getPackagerRunAs().then(packagerRunAs => {
+                        if (packagerRunAs === PackagerRunAs.EXPONENT
```

Comment:
We might also want similar logic in /src/common/exponent/exponentPlatform.ts, unless that is already part of starting the packager.

In fact, perhaps we should always just call `this.remoteExtension.startPackager()` here, and move the "Is one already running, and is it the right one? Maybe stop and restart" logic into the extension process rather than here in the debugger.